oqs-template/generate.yml

```diff
@@ -604,20 +616,44 @@ sigs:
                           'pretty_name': 'RSA3072',
                           'oid': '1.3.9999.3.3',
                           'code_point': '0xfe0d'}]
+      -
+        name: 'falcon512_padded'
```

Comment:
What about leaving this "_" away? That then would at least mimick the sphincs naming. My preference of course would be to use the name "falcon_padded_512". Should be possible as long as "oqs_meth" is kept as-is (if https://github.com/open-quantum-safe/liboqs/pull/1710 lands unchanged). But as you begun to take responsibility for this file and a first downstream with a reading on this (profiling -- both greatly appreciated!) you get to decide.

Reply:
I've switched to `falconpadded512`/ `falconpadded1024` in the latest push to mimic the current usage of `falcon512` / `falcon1024` (sans underscore).